src/index.ts

```diff
@@ -11,7 +12,17 @@ export {AudioPlayer as Plugin};
 export {VERSION, NAME};
 
 const pluginName = 'audioPlayer';
+registerPlugin(pluginName, AudioPlayer);
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-registerPlugin(pluginName, AudioPlayer);
+window.__kalturaplayerdata = {
```

Comment:
i dont like injecting into config like this because first of all, it will break if config structure ever changes, and second - i dont know what happens in case there are multiple players on the page